README.md

```diff
@@ -3,383 +3,17 @@
 ![CI](https://github.com/TuringLang/MCMCChains.jl/workflows/CI/badge.svg?branch=master)
 [![codecov](https://codecov.io/gh/TuringLang/MCMCChains.jl/branch/master/graph/badge.svg?token=TFxRFbKONS)](https://codecov.io/gh/TuringLang/MCMCChains.jl)
 [![Coverage Status](https://coveralls.io/repos/github/TuringLang/MCMCChains.jl/badge.svg?branch=master)](https://coveralls.io/github/TuringLang/MCMCChains.jl?branch=master)
+[![][docs-stable-img]][docs-stable-url] 
+[![][docs-latest-img]][docs-latest-url]
 
 Implementation of Julia types for summarizing MCMC simulations and utility functions for diagnostics and visualizations.
 
-## Example
```

Comment:
I have started with it but got scared of a bit by the introduction of yet another opportunity for outdated docs. Arguably, DataFrames has gotten quite popular even though it [doesn't show examples in its README](https://github.com/JuliaData/DataFrames.jl/blob/main/README.md). If you don't mind, I just add a basic `plot(chn)` and link it to the docs. This will be included in the next commit.